rundeck_exporter.py

```diff
@@ -525,8 +586,21 @@ def collect(self):
                 else:
                     projects = self.request(endpoint)
 
+            job_option_keys = []
+            if self.args.rundeck_projects_executions_export_job_options:
+                if self.args.rundeck_projects_executions_job_option_labels:
+                    job_option_keys = self.args.rundeck_projects_executions_job_option_labels
+                else:
+                    job_option_keys = self.get_all_job_option_keys(projects)
+
+            job_option_labels = [
+                f'opt_{re.sub(r"[^a-zA-Z0-9_]", "_", key)}'
+                for key in job_option_keys
+            ]
```

Comment:
I think we can find a way to do this without calling `get_all_job_option_keys`, as it is expensive and essentially does the same thing as `get_project_executions`.
Perhaps some logic could be added to the `get_project_executions` method, and some class instance variables could be used.